### ISSUE TITLE
Support for Django 2.0

### ISSUE THREAD

**Issue by GasparPizarro** [tanwanirahul/django-batch-requests#15]
Are there plans for supporting Django 2.0 in the future?

**Comment by tanwanirahul** [comment]
@GasparPizarro Yes, I would love to. If you have extended this for Django 2.0, it would make sense to review and merge.
